validator/src/bootstrap.rs

```diff
@@ -1125,6 +1129,10 @@ mod with_incremental_snapshots {
         peer_snapshot_hashes
     }
 
+    /// Map full snapshot hashes to a set of incremental snapshot hashes.  Each full snapshot hash
+    /// is treated as the base for its set of incremental snapshot hashes.
+    type GoldenSnapshotHashes = HashMap<(Slot, Hash), HashSet<(Slot, Hash)>>;
```

Comment:
I don't love the introduction of the new term, "golden snapshot".  What makes it golden?  These are local snapshots?

Reply:
known snapshot hashes?  We specify `--known-validators`, not `--golden-validators` after all.  I'll look at the full PR now, sorry last night my jet lagged brain was only good for some low-ROI bike shedding :)

Reply:
Done! Renamed "golden" back to "known".